rest/route/user.go

```diff
@@ -364,3 +364,90 @@ func (h *userPermissionsGetHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 	return gimlet.NewJSONResponse(permissions)
 }
+
+type userRolesPostHandler struct {
+	sc     data.Connector
+	rm     gimlet.RoleManager
+	userID string
+	roles  []string
+}
+
+func makeModifyUserRoles(sc data.Connector, rm gimlet.RoleManager) gimlet.RouteHandler {
+	return &userRolesPostHandler{
+		sc: sc,
+		rm: rm,
+	}
+}
+
+func (h *userRolesPostHandler) Factory() gimlet.RouteHandler {
+	return &userRolesPostHandler{
+		sc: h.sc,
+		rm: h.rm,
+	}
+}
+
+func (h *userRolesPostHandler) Parse(ctx context.Context, r *http.Request) error {
+	vars := gimlet.GetVars(r)
+	h.userID = vars["user_id"]
+	if h.userID == "" {
+		return errors.New("no user found")
+	}
+	roles := []string{}
```

Comment:
I think you can ommit this. and just &h.roles in the next line.

Reply:
done